cmd/sdk-server/main.go

```diff
@@ -241,7 +241,10 @@ func runGateway(ctx context.Context, grpcEndpoint string, mux *gwruntime.ServeMu
 	}
 
 	if err = sdk.RegisterSDKHandler(ctx, mux, conn); err != nil {
-		logger.WithError(err).Fatal("Could not register grpc-gateway")
+		logger.WithError(err).Fatal("Could not register sdk grpc-gateway")
+	}
+	if err = sdkalpha.RegisterSDKHandler(ctx, mux, conn); err != nil {
```

Comment:
Couldn't add the beta SDK in here, as with no services, no `RegisterSDKHandler` function gets generated for the beta SDK.